src/main/java/util/JsonExceptionMapper.java

```diff
@@ -0,0 +1,56 @@
+package util;
+
+import javax.annotation.PostConstruct;
+import javax.ejb.Singleton;
+import javax.ejb.Startup;
+import javax.json.Json;
+import javax.json.JsonBuilderFactory;
+import javax.json.JsonObject;
+import javax.ws.rs.WebApplicationException;
+import javax.ws.rs.core.Response;
+import java.util.HashMap;
+import java.util.Map;
+
+@Startup
+@Singleton
+public class JsonExceptionMapper {
+
+	private static Map<String, Object> config;
+
+	@PostConstruct
+	public void setup(){
+		config = new HashMap<>();
```

Comment:
Waar wordt het config object voor gebruikt?

Reply:
prettifyen van JSON, kan ook null zijn. zie code in setup.